tests/invalid/integer/incomplete-oct.toml

```
@@ -0,0 +1 @@
+incomplete-oct = 0c
```

I think this should be:

```suggestion
incomplete-oct = 0o
```

?

---

Good catch!